### DISCUSSION TITLE
Does OTBR support dynamically switching infrastructure network interfaces without restarting the otbr-agent process?

### DISCUSSION THREAD

**Discussion by xiaojiang2017** [openthread/openthread#8771] [Q&A] [answered]
Hi experts,
According to the guide [2. Setup OTBR](https://openthread.io/codelabs/openthread-border-router#1), the infrastructure network interfaces will be specified when otbr-agent process startup.
But in multi-WAN devices, [mwan3 ](https://openwrt.org/docs/guide-user/network/wan/multiwan/mwan3#mwan3_multi_wan_load_balancingfailover)is usually used for multi-WAN management. Does this affect otbr-agent?
And if the infrastructure network is switched from wifi to ethernet, do I need to restart otbr-agent to specific `backboneIfName`?

**Comment by jwhui** [comment, marked as answer]
Take a look at:
- https://github.com/openthread/ot-br-posix/pull/1353

Note that if `otbr-agent` selects a different infrastructure link, it will quit and expect the system to start the process again.
